src/languages/es.json

```diff
@@ -0,0 +1,118 @@
+        "button_make": "Generar llave",
+        "create_account": "Crear cuenta nueva de TestNet",
+        "button_open": "Abrir tu monedero",
+        "guide_title": "Instrucciones para mandar BOScoin con tu cuenta",
+        "guide": {
+            "step1_1": "Crea una nueva llave por ",
+            "step1_2": " botón de arriba",
+            "step2": "Escriba abajo las7 dos llaves y guárdelas de manera segura (¡Muy Importante!)",
+            "step3_1": "Abre tu cuenta por ",
+            "step3_2": " botón de arriba",
+            "step4": "Introduce la semilla de tu dirección en el cuadro blanco",
+            "step5_1": "Presiona ",
+            "step5_2": " el botón y verás tu cuenta",
+            "step6_1": "Presiona ",
+            "step6_2": " el botón e introduce la dirección pública del destinatario en el cuadro blanco",
+        },
+        "error": {
+            "fail_create_account": "Fallo al crear la cuenta."
+        },
+        "test_mode": "Esto es un<br><strong>Monedero web de TEST!</strong>",
+    },
+    "login_view": {
+        "header": "Introduce tu semilla",
+        "guide_line_1": "Por favor, introduce tu semilla secreta para acceder a tu cuenta.",
+        "guide_line_2": "Por favor, guarda la semilla secreta de forma segura, de lo contrario la seguridad de tu monedero podría ser comprometida.",
+        "error": {
+            "invalid_secret_seed": "Semilla secreta inválida o balance insuficiente (1 BOScoin mínimo)."
+        },
+    },
+    "send_coin": {
+        "input_recipient_address": "Introduce la dirección pública del destinatario",
+        "input_amount": "Introduce la cantidad que deseas enviar",
+        "transaction_fee": "Comisión de la transacción",
+        "total_will_be_sent": "Un total de {amount} BOScoins serán enviados al destinatario",
+        "error": {
+            "public_address_null": "Introduce la dirección pública del destinatario",
+            "transaction_amount_null": "Introduce la cantidad para la transacción",
+            "transaction_amount_zero": "Introduce la cantidad para la transacción",
+            "not_enough_balance": "Balance Insuficiente",
+            "incorrect_public_address": "Dirección pública incorrecta",
+            "minimum_balance": "El balance mínimo tras la transacción ha de ser de almenos {minimum_balance} BOS.",
+            "decimal_limit": "Solo puedes introducir 7 decimales.",
+        },
+    },
+    "key_generator": {
+        "header": "Pulsa para generar una nueva pareja de claves",
+        "description_line1": "Esta herramienta no te creará una cuenta.",
+        "description_line2": "Deberás mandar una operación del tipo create_account a la siguiente",
+        "description_line3": "dirección con un balance inicial para que pueda existir. ",
+        "description_line4": "Guarda tu semilla de forma segura.",
+        "warn_line1": "Por facor, copia tu dirección pública y la semilla secreta y guardaló en algún lugar seguro.",
```

Comment:
I think you entered wrong.

Wrong input  facor -> favor

Por favor, copia tu dirección pública y la semilla secreta y guardaló en algún lugar seguro